MANIFEST.in

```diff
@@ -8,3 +8,4 @@ recursive-exclude * *.swp
 recursive-exclude * __pycache__
 recursive-exclude * *.db
 recursive-exclude * local_settings.py
+recursive-exclude tests *
```

Comment:
I think we can add `recursive-exclude openwisp_notifications/tests *` too

Reply:
I don't think this will be a good idea. 
In past I ran, tests of firmware-upgrader from openwisp-monitoring to check whether notifications for upgrade were being created or not. :sweat_smile:

Reply:
This change doesn't prevent you from running the tests, it just doesn't include it in the package that you install using `pip`